### ISSUE TITLE
Clean up issues' tags

### ISSUE THREAD

**Issue by ashfurrow** [Moya/Moya#538]
Most of first page of results in the [Issues](https://github.com/Moya/Moya/issues) page don't have any labels. Those issues should be labeled appropriately. I'll get to it eventually but if another Moya contributors is keen, I welcome any help with it.

We currently have the following [GitHub labels](https://github.com/Moya/Moya/labels):

<img width="352" alt="screen shot 2016-07-12 at 8 22 25 pm" src="https://cloud.githubusercontent.com/assets/498212/16787938/5edf7e54-486e-11e6-90fb-78e4a268325a.png">

Seems like a fine list, but if anyone has changes they'd like to see, this is a good place to have that discussion too.

**Comment by sunshinejr** [comment]
@ashfurrow because we've closed milestones(#539), do you think we should add **Documentation** issue tag, or just try to use appropriate one from the existing tags?

**Comment by ashfurrow** [comment]
I've created the documentation label, milestones look good, thanks!

**Comment by ashfurrow** [comment]
Okay, think I got this done. I also closed lots of old issues, asked about some I wasn't sure of. 

@Moya/contributors We've still got plenty of GitHub issues, especially lots of [questions](https://github.com/Moya/Moya/issues?q=is%3Aissue+is%3Aopen+label%3Aquestion). No pressure at all – if you have some free time to help out, that'd be awesome! But don't worry about it. In any case, have a great weekend!
